src/data/real/basic.lean

```diff
@@ -125,9 +148,7 @@ instance : has_trivial_star ℝ   := ⟨λ _, rfl⟩
 /-- Coercion `ℚ` → `ℝ` as a `ring_hom`. Note that this
 is `cau_seq.completion.of_rat`, not `rat.cast`. -/
```

Comment:
Should this be updated?

Reply:
It's probably best to standardize on one spelling, so let's say the most generic which is `algebra_map` (right?)

Reply:
This is now moot, `real.of_rat` is gone.

Reply:
`padic.of_rat` is now also gone.